src/NzbDrone.Core/MediaFiles/EpisodeImport/ImportApprovedEpisodes.cs

```diff
@@ -92,6 +92,7 @@ public List<ImportResult> Import(List<ImportDecision> decisions, bool newDownloa
                     episodeFile.Size = _diskProvider.GetFileSize(localEpisode.Path);
                     episodeFile.Quality = localEpisode.Quality;
                     episodeFile.MediaInfo = localEpisode.MediaInfo;
+                    episodeFile.Series = localEpisode.Series;
```

Comment:
Instead of assigning the series here we could conditionally use `Path` or `RelativePath` when building the path in webhooks, if we have a path `Path` (not null or empty) use it, otherwise use the relative path.

Reply:
`Path` was never stored in the DB, in it's only present in some scenarios, such as new imports when a series may not be linked to the file yet.

Reply:
I was definitely leaning towards this fix, but it being the source path is 🤯 thanks for checking!